src/utils/time.ts

```diff
@@ -13,5 +13,8 @@ export const nanoToSeconds = (nano: number) => {
  * @returns days
  */
 export const secondsToDays = (seconds: number) => {
+  if ((seconds / (3600 * 24)) < 1) {
+    return seconds;
+  }
```

Comment:
no! this function only cares about `secondsToDays` it will return days no matter what

Reply:
ohhh ok!